src/client/options/test.rs

```diff
@@ -239,7 +240,9 @@ fn run_test(test_file: TestFile) {
                 });
 
                 if !is_unsupported_host_type {
-                    let options = ClientOptions::parse(&test_case.uri).unwrap();
+                    let options = RUNTIME
```

Comment:
spec test functions can't be async until `run_spec_tests` is updated. I figured doing so would be out of scope for this work, so for now I just block on any async methods.

Reply:
That was one of the things I ran into when I first tried to convert the entire API in one go.